### ISSUE TITLE
limel-select pulls focus away to it when its values have been updated

### ISSUE THREAD

**Issue by sampsonbryce** [Lundalogik/lime-elements#647]
## Current behavior

When limel-select gets an update it pulls focus to it. This can cause some unexpected behavior because changing unrelated inputs causes the focus to be pulled to the limel-select. The offending code can be found in select.ts

```javascript
    public componentDidUpdate() {
        if (this.menuOpen) {
            this.setMenuFocus();
        } else {
            this.setTriggerFocus();
        }
    }
```
The limel-select element could be rerendered for a number of reason including the parent being rerendered due to an unrelated change. In my use case editing a limel-input-field caused focus to be pulled to the limel-select element.

## Expected behavior

Focus should not be pulled to limel-select unless the user clicks on the element

## Environment

- **lime-elements** version: 26.11.2
- Framework used: StencilJS 1.10.0
- Logs: N/A

**Comment by hannahu** [comment]
I'm pretty sure this will be solved by the fix in https://github.com/Lundalogik/crm-feature/issues/1019

**Comment by sampsonbryce** [comment]
The fix in https://github.com/Lundalogik/crm-feature/issues/1019 doesn't address the offending code as shown above. I experienced the issue on select elements not on picker elements as well. Can you explain some more how the changes will fix this issue? Maybe I'm not understanding the code changes correctly?

**Comment by hannahu** [comment]
> The fix in [Lundalogik/crm-feature#1019](https://github.com/Lundalogik/crm-feature/issues/1019) doesn't address the offending code as shown above. I experienced the issue on select elements not on picker elements as well. Can you explain some more how the changes will fix this issue? Maybe I'm not understanding the code changes correctly?

I read too fast and takes back my comment :) This is another issue!
